### PR TITLE
Add "Contact us" section to README

### PR DESCRIPTION
**Pull request by mhmdanas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,3 +167,10 @@ you don't have to recompile Numbat to see your changes.
 Alternatively, you can create a symlink from `~/.config/numbat/modules` to the `numbat/modules/`
 folder in the repository (see [this page](https://numbat.dev/doc/cli-customization.html#module-paths)
 for the standard paths on other operating systems.
+
+## Contact us
+
+To contact us, either [open a GitHub issue](https://github.com/sharkdp/
+numbat/issues/new/ choose) or [discussion](https://github.com/sharkdp/numbat/
+discussions), or pop into `#numbat` on [Libera.Chat](https://libera.chat) ([link
+to webchat](https://web.libera.chat/gamja/#numbat)).
```
Comment (mhmdanas) [inline README.md:176]: Do you think this section should be added to the website as well?

File: README.md [hunk 2]
```diff
@@ -167,3 +167,10 @@ you don't have to recompile Numbat to see your changes.
 Alternatively, you can create a symlink from `~/.config/numbat/modules` to the `numbat/modules/`
 folder in the repository (see [this page](https://numbat.dev/doc/cli-customization.html#module-paths)
 for the standard paths on other operating systems.
+
+## Contact us
+
+To contact us, either [open a GitHub issue](https://github.com/sharkdp/
+numbat/issues/new/ choose) or [discussion](https://github.com/sharkdp/numbat/
+discussions), or pop into `#numbat` on [Libera.Chat](https://libera.chat) ([link
```
Comment (eminence) [inline README.md:175]: Is anyone actually in this IRC channel?  Looks empty at the moment :)

Comment (mhmdanas) [reply README.md:175]: I have it on autojoin so whenever I'm online I'll be there. :)

**Review by sharkdp — APPROVED** [review]
Thank you

**Pull request merged**